lib/std/fs/file.zig

```diff
@@ -927,6 +927,12 @@ pub const File = struct {
     /// Reads all the bytes from the current position to the end of the file.
     /// On success, caller owns returned buffer.
     /// If the file is larger than `max_bytes`, returns `error.FileTooBig`.
+    /// `max_bytes` is not "number of bytes to read", but a maximum.
+    ///
+    /// Note:
+    /// It's imperative to choose a reasonable value for `max_bytes`. The least desirable value to
```

Comment:
"imperative" is a word in the realm of morality and authority. Instead, just explain how it works.